infrastructure/lambda/public-api.yaml

```diff
@@ -119,6 +119,143 @@ paths:
         contentHandling: "CONVERT_TO_TEXT"
         type: "aws_proxy"
 
+  /.well-known/jwks.json:
+    get:
+      operationId: getWellKnownJwksJson
+      summary: Retrieve the encryption public keys for JWTs issued by `POST /userinfo`.
```

Comment:
It's not used by POST /userinfo. The encryption keys are used to send data (in the form of a JWT) to the CRI's /authorize which are handled by /session on the Private API